prow/daily-e2e-cluster_wide-auth.sh

```diff
@@ -33,9 +33,11 @@ fi
 
 # Exports $HUB, $TAG, and $ISTIOCTL_URL
 source greenBuild.VERSION
+echo "Preparing to run daily-e2e-cluster_wide-auth-test"
```

Comment:
Remove this line. Not only is this redundant, the cluster wide e2e that is using default proxy will also call this script and that would be false information to print.